wp/README.md

```diff
@@ -417,6 +417,13 @@ The various options are:
   be denoted in bytes. By default, the size of the stack is 0x800000, which is
   8Mbs.
 
+- `--op-uninterp=[plus|sub|mul|udiv|et.seq.] or some comma-delimited combination`.
```

Comment:
I'm not sure if we should have a default when calling this flag with no arguments. I'm expecting that no arguments means that we're not making anything uninterpreted. 

Can you also add a quick explanation and/or example explaining what uninterpreted means?

Reply:
That makes sense. I'll make the default the empty-list.
The example/explaination idea is good, since the concept is not intuitive to most.